### PR TITLE
When adding a collaborator via the API, add created_by/updated_by

### PR DESCRIPTION
**Pull request by suricactus**
In addition, prevent attempts to add other `User` subclasses as project
collaborators

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-4534 Ensure API endpoints for adding collaborators and members works](https://app.clickup.com/t/2192114/QF-4534)

File: docker-app/qfieldcloud/core/serializers.py [hunk 1]
```diff
@@ -188,20 +188,35 @@ class Meta:
...
+            "updated_by",
+            "created_at",
+            "updated_at",
+        )
 
 
 class OrganizationMemberSerializer(serializers.ModelSerializer):
+    organization = serializers.StringRelatedField()
     member = serializers.StringRelatedField()
     role = serializers.CharField()
```
Comment (faebebin) [inline docker-app/qfieldcloud/core/serializers.py:210]:
```suggestion
```

**Review by faebebin — CHANGES_REQUESTED** [review]
Looks good

One little cleanup suggestion

else great, thanks

File: docker-app/qfieldcloud/core/serializers.py (hunk 1 above)
Comment (suricactus) [reply docker-app/qfieldcloud/core/serializers.py:210]: Ah good spot.

**Review by faebebin — APPROVED** [review]
LGTM

just the one linting fix, but else good
thanks

**Comment by suricactus** [comment]
The linting was fixed on `master`. Thanks!

**Pull request merged**